.circleci/config.yml

```diff
@@ -21,9 +21,13 @@ jobs:
           paths:
             - ~/.npm
       - run:
-          name: "Build"
+          name: "Test"
```

Comment:
Didn't make sense that the test command was named "Build" so changed it accordingly.